go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v0.1.0
 	github.com/go-openapi/spec v0.19.2
+	github.com/go-openapi/validate v0.18.0 // indirect
```

Comment:
This and the change to `go.sum` seem to be independent of the test change. Can this be a separate commit?

Reply:
Yes. Separated.